react/Dropdown/Dropdown.sketch.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dropdown from './Dropdown';
+import styles from './Dropdown.sketch.less';
+
+const Container = ({ children }) => (
+  <div style={{ maxWidth: '520px' }}>
```

Comment:
I feel like we should capture this boilerplate somewhere to be shared between all sketch files. Thoughts?

Reply:
Yep, makes sense. At some point, it's probably worth discussing whether this rule should be applied to the fields themselves, since we don't generally want fields to get wider than this, regardless of the surrounding layout.

Reply:
Just pushed a refactor 👍